File: Leetcode/1672/Linear_Search_Algo/main.py
# using Python old
class Solution(object):
    def maximumWealth(self, accounts):
        """
        :type accounts: List[List[int]]
        :rtype: int
        """
        max = 0

        for customer in accounts:
            transactions = 0
            for transaction in customer:
                transactions = transactions + transaction
            if (max < transactions):
                max = transactions
        
        return max

File: Leetcode/1672/Linear_Search_Algo/main.swift
class Solution {
    func maximumWealth(_ accounts: [[Int]]) -> Int {
        var max: Int = 0
        for customer in accounts {
            var transactions: Int = 0
            for transaction in customer {
                transactions += transaction
            }
            if (max < transactions) {
                max = transactions
            }
        }
    return max        
    }
}